### PR TITLE
users: typos, formating, context

### PR DESCRIPTION
**Pull request by Zyzto**
* typo in line 11,10
* formatting in 10
* changed to action context in line 4

### CODE REVIEW

**Comment by spencerjunraid** [comment]
Thx.

File: users.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
 Add User=إضافة مستخدم
 Add=إضافة
 Click to select PNG file=إضغط لإختيار صورة
```
Comment (alhamesrami) [inline users.txt:3]: انقر لاختيار صورة ملف PNG
I already changed this, have we saved the version I submitted?

File: users.txt [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-Custom image=صورة خاصة
+Custom image=تخصيص الصورة
 Description=وصف
 Do not use reserved names=لا تستخدم أسماء محجوزة
 Drag-n-drop a PNG file or click the image at the left=اسحب و ضع ملف صورة او اضغط على الصورة على اليسار
```
Comment (alhamesrami) [inline users.txt:7]: اسحب وضع ملف صورة أو انقر على الصورة على اليسار
This as well

**Review by alhamesrami — COMMENTED** [review]
Seems that older version is coming back, 2 comments here

**Comment by Zyzto** [comment]
I missed them, and sorry for delay I am not getting notifications from email on my phone for some while.

I will fix them a bit later , I am in reviewing apps.text before first PR for review

**Pull request merged**